index.html

```diff
@@ -200,58 +205,75 @@ <h3 class="about-card-title">Skills</h3>
             </div>
         </section>
 
-
         <!-- Contact me section -->
         <section id="contact-section">
-            <h2 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat
-                please get
-                in touch.</h2>
-            <div class="form-container">
-                <form class="contact-form" action="https://formspree.io/f/mvonlebo" method="POST">
-                    <div class="form-group">
-                        <label for="full-name"></label>
-                        <input type="text" id="full-name" name="full-name" placeholder="Full Name" maxlength="30"
-                            required>
-                    </div>
-                    <div class="form-group">
-                        <label for="email"></label>
-                        <input type="email" id="email" name="email" placeholder="Email address" required>
-                    </div>
-                    <div class="form-group">
-                        <label for="message"></label>
-                        <textarea id="message" name="message" placeholder="Hey, I'd like to connect with you about..."
-                            maxlength="500" required></textarea>
-                    </div>
-                    <div class="form-group">
-                        <input type="submit" value="Get in touch" class="contact-btn">
-                    </div>
+            <div class="contact-container">
+                <div class="grid-container">
+                    <h2 class="contact-text grid-item">I'm always interested in hearing about new projects, so if you'dlike to chat please get in touch.</h2>
+                    <div class="form-container grid-item">
+                        <form class="contact-form" action="https://formspree.io/f/mvonlebo" method="POST">
+                            <div class="desktop-fields">
+                                <div class="form-group">
+                                    <label for="firstname"></label>
+                                    <input type="text" id="first-name" name="first-name" placeholder="First Name" maxlength="30" required>
+                                </div>
+                                <div class="form-group">
+                                    <label for="lastname"></label>
+                                    <input type="text" id="last-name" name="last-name" placeholder="Last Name" maxlength="30" required>
+                                </div>
```

Comment:
- [ ] I also think you could make some adjustments to your design here. :+1:  Looking at he screenshots below, you can see that your `first name` and `last name` input are not properly stretched and aligned like the Figma design, I'll suggest you update the styling here. :+1: 
 - [ ] Also the button is positioned at the left and not at the center, following the Figma design, I'll suggest you update your design accordingly as being able to replicate a given design is a very valuable skill for a front-end developer. :smile: :wave: 

- - Your design:
![Selection_038](https://github.com/RileyManda/portfolio-website/assets/107632780/261034cb-e210-4aee-8090-baf36b027c8d)

- - Figma design:
![Screenshot from 2023-05-11 23-17-42](https://github.com/RileyManda/portfolio-website/assets/107632780/59df57bb-0fc7-40ee-9d5e-f6967dca5f3c)

Reply:
I have completed iterations on the Desktop Version of the Portfolio Project as requested by the reviewer.
Here are the iterations i have made to the project: 

- Replaced first name and last name form fields with full name field as this is optional according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md): 
- Fixed the form button alignment-positioning it to the left.
- Centered the card tags.
- Updated the About Section and aligned text to centered to match the UX Design.
- Aligned cards to center in desktop view
In addition: 

- I have aligned the card container to center for desktop view.

- Added nth element CSS to re-arrange social icons to match desktop view._**(Mobile and Desktop icons alignment is different according to the design).**_

In general, The project has been designed to meet the following requirements:

- Used media queries to ensure the website displays correctly on desktop screens [(min-width: 768px)](https://github.com/RileyManda/portfolio-website/blob/desktop-version/style.css)
-   Made sure the components look great and function well on different devices.
-   Used grid layout and flex layout to align elements.
-   Made sure that there are no linter errors
-   Attempted to replicate the UX design as much as possible.
-  Ensured there are no linter errors in the project.
-  Used Correct git-flow.
- Documented my work in a professional manner according to requirements,
[(https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).].
- Included a descriptive [Readme](https://github.com/RileyManda/portfolio-website/blob/desktop-version/README.md) file.
- Used descriptive commit messages
- Followed the recommended HTML&CSS best [practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Please review the changes and let me know if there are any issues or suggestions for improvement. I look forward to your feedback. 

Thank you! 
Best regards,🙏